pipeline/05-finalize.R

```diff
@@ -9,6 +9,7 @@
 suppressPackageStartupMessages({
   library(arrow)
   library(aws.s3)
+  library(aws.ec2metadata)
```

Comment:
As we discovered in https://github.com/ccao-data/model-res-avm/pull/26, this package is necessary in order to allow the `aws.s3` package to authenticate using credentials in an ECS environment.